### PR TITLE
[FIX] 14.0 pos_report_session_summary: do not crash if statement data is missing

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

File: pos_report_session_summary/views/report_session_summary.xml [hunk 1]
```diff
@@ -122,11 +122,21 @@
...
                             <tbody>
                                 <tr t-foreach="statement.line_ids" t-as="line">
-                                    <td><span t-field="line.date" /></td>
-                                    <td><span t-field="line.name" /></td>
-                                    <td><span t-field="line.ref" /></td>
-                                    <td><span t-field="line.partner_id" /></td>
-                                    <td><span t-field="line.account_id" /></td>
+                                    <td><t t-if="'date' in line"><span
+                                                    t-field="line.date"
+                                                /></t></td>
```
Comment (ivantodorovich) [inline pos_report_session_summary/views/report_session_summary.xml]:
```suggestion
                                    <td><span t-if="'date' in line" t-field="line.date" /></td>
```
For readability, you can add `t-if` directive directly to the `span`

**Review by FernandoRomera — APPROVED** [review]
Review functional, OK.

**Review by AntoniRomera — APPROVED** [review]

**Comment by FernandoRomera** [comment]
@ivantodorovich Can you merge this PR?

**Review by ivantodorovich — APPROVED** [review]
@PierrickBrun  Thanks, looks good

Could you just squash your fixup commit, please 🙏🏻 ?

**Comment by PierrickBrun** [comment]
@ivantodorovich done

**Pull request merged**